### PR TITLE
Master | LPS-159175, LPS-163037

### PR DESCRIPTION
**Pull request by Hanlf**
https://issues.liferay.com/browse/LPS-159175
https://issues.liferay.com/browse/LPS-163037

### CODE REVIEW

**Comment by Hanlf** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :eyes:

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBArticle.testcase [hunk 1]
```diff
@@ -352,6 +352,48 @@ definition {
...
+		KBArticle.viewInfoPanelVersions(
+			entryVersion = "Version 1",
+			userName = "Test Test");
+
+		KBAdmin.openKBAdmin(siteURLKey = "guest");
+
+		KBArticle.editCP(
+			kbArticleContentEdit = "Knowledge Base Article Content Edit",
+			kbArticleTitle = "Knowledge Base Article Title",
+			kbArticleTitleEdit = "Knowledge Base Article Title Edit");
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBArticle.testcase:385]: @Hanlf can you create a JSON macro for updating a KB article here?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBArticle.testcase [hunk 2]
```diff
@@ -352,6 +352,48 @@ definition {
 			kbArticleTitle = "Knowledge Base Article Title");
 	}
 
+	@description = "This test ensures that the informations related to the KB entry can be viewed."
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBArticle.testcase:359]: information

**Comment by austinchiang** [comment]
Please resend after fixing the changes, thx

**Pull request closed without merging**